### PR TITLE
Feature: Pre-select organization in dropdown from an org page

### PR DESCRIPTION
**Pull request by kmanaseryan**
Related issue:
https://app.zenhub.com/workspaces/datagov-multi-tenant-ckan-5cf12b601437ff1e943f0aa2/issues/gsa/ckanext-dcat_usmetadata/109

Demo:
https://www.loom.com/share/88c064f21bab498480769a6baae6d6f8

### CODE REVIEW

File: cypress/integration/required-metadata.spec.js [hunk 1]
```diff
@@ -104,6 +104,15 @@ describe('Required Metadata Page', () => {
         cy.contains('my default title');
       });
   });
+
+  it('Organization is pre-selected in the dropdown if the form is loaded from organization page', () => {
+    cy.login();
+    cy.visit('/organization/test-123');
```
Comment (anuveyatsu) [inline cypress/integration/required-metadata.spec.js:110]: @kmanaseryan I would expect `?group=...` parameter here

Comment (kmanaseryan) [reply cypress/integration/required-metadata.spec.js:110]: @anuveyatsu it visits organization page then clicks on the button "Add dataset" below

Comment (anuveyatsu) [reply cypress/integration/required-metadata.spec.js:110]: oh, you are right @kmanaseryan

**Review by anuveyatsu — APPROVED** [review]
👍

**Pull request merged**